core/domain/services/assets/EspressoEditorAssetManager.php

```diff
@@ -45,7 +45,17 @@ public function addAssets()
      */
     private function registerJavascript()
     {
-        $this->addJs(self::JS_HANDLE_EDITOR, [CoreAssetManager::JS_HANDLE_JS_CORE])->setRequiresTranslation();
+        $dependencies = defined('SCRIPT_DEBUG')
```

Comment:
I am a kind of person who sets `SCRIPT_DEBUG` to false when I want to test things for production 🙈 
```suggestion
        $dependencies = defined('SCRIPT_DEBUG') && SCRIPT_DEBUG
```

Reply:
oh oops... I swear that's how it was in my original version that got lost, but I forgot to do that this time around